aws-params.yaml

```diff
@@ -5,4 +5,5 @@ metaphlan_db: "s3://biobakery-databases/metaphlan_databases/"
 humann_bowtie_db: "s3://biobakery-databases/humann_databases/chocophlan"
 humann_protein_db: "s3://biobakery-databases/humann_databases/uniref"
 humann_utility_db: "s3://biobakery-databases/humann_databases/utility_mapping"
-filepattern: "*_L00{1,2,3,4}_R{1,2}_001.fastq.gz"
\ No newline at end of file
+filepattern: "*.fastq.gz"
```

Comment:
We should probably leave the other on - *.fastq.gz is much easier to remember...